abci/Cargo.toml

```diff
@@ -27,6 +27,7 @@ tenderdash-proto = { version = "0.11.0-dev.4", default-features = false, path =
 tracing = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3", optional = true, default-features = false }
 thiserror = "1.0.39"
+serde = { version = "1.0.152", features = ["derive"] }
```

Comment:
Can't find where it's needed

Reply:
yeah but I see no configuration layer here in the library, I think matching config input to the enum should be done where the configuration happens;

and it could be two functions again, just both return `dyn Server` instead of this enum; otherwise with this (de)serialization it looks like abci library takes something it's not responsible for